test/platform/azure/index.spec.ts

```diff
@@ -369,7 +390,7 @@ describe('platform/azure', () => {
     });
   });
 
-  describe('getBranchStatus(branchName, requiredStatusChecks)', () => {
+  describe('gsetBranchStatus(branchName, requiredStatusChecks)', () => {
```

Comment:
Typo

Reply:
Can't commit 🤔

Reply:
@JamieMagee you can create branches within the main repo from now on if you like. easier for us to edit

Reply:
Sure, will do in future :smile: